webview-ui/src/features/deep-research/GetStarted.tsx

```diff
@@ -0,0 +1,278 @@
+import { useCallback, useEffect, useState } from "react"
+import { useForm, FormProvider, Controller } from "react-hook-form"
+import { zodResolver } from "@hookform/resolvers/zod"
+import { CircleBackslashIcon, RocketIcon } from "@radix-ui/react-icons"
+import { ChevronsUpDown, ChevronsDownUp, BrainCircuit } from "lucide-react"
+
+import { cn } from "@/lib/utils"
+import {
+	Button,
+	Slider,
+	AutosizeTextarea,
+	Input,
+	Collapsible,
+	CollapsibleContent,
+	CollapsibleTrigger,
+} from "@/components/ui"
+
+import { useResearchSession } from "./useResearchSession"
+import { useProvider } from "./useProvider"
+import { ResearchSession, researchSessionSchema } from "./types"
+import { Providers } from "./Providers"
+import { Models } from "./Models"
+
+export const GetStarted = () => {
+	const [isProvidersOpen, setIsProvidersOpen] = useState(false)
+	const { setSession } = useResearchSession()
+	const { provider, providers, setProviderValue } = useProvider()
+
+	const form = useForm<ResearchSession>({
+		resolver: zodResolver(researchSessionSchema),
+		defaultValues: {
+			providerId: provider?.providerId ?? "",
+			modelId: "o3-mini",
```

Comment:
Is this a better default than o3-mini-high?

Reply:
Fixed.